### PR TITLE
docs: Fix spelling mistakes in apis.ipynb

### PR DESCRIPTION
**Pull request by frgul006**
- Description: Fix spelling mistakes in apis.ipynb
  - Issue: [#9910](https://github.com/langchain-ai/langchain/issues/9910)

### CODE REVIEW

**Review by leo-gan — COMMENTED** [review]
a nit

File: docs/extras/use_cases/apis.ipynb [hunk 1]
```diff
@@ -155,10 +155,10 @@
     "https://www.klarna.com/us/shopping/public/openai/v0/api-docs/\n",
     "```\n",
     "\n",
-    "* The prompt then tells the LLM to use the API spec wiith input question:\n",
+    "* The prompt then tells the LLM to use the API spec with input question:\n",
     "\n",
     "```\n",
-    "Use the provided API's to respond to this user query:\n",
+    "Use the provided APIs to respond to this user query:\n",
```
[the remark this answers is not in the document]
Comment (baskaryan) [reply docs/extras/use_cases/apis.ipynb:161]: think there's no ' since it's not possessive (or a contraction)

**Comment by baskaryan** [comment]
thanks @frgul006!

File: docs/extras/use_cases/apis.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (leo-gan) [reply docs/extras/use_cases/apis.ipynb:161]: Hmm... I've tried two spellcheckers, both insisted on `APIs`.

[the remark this answers is not in the document]
Comment (baskaryan) [reply docs/extras/use_cases/apis.ipynb:161]: isn't that what it was changed to?

[the remark this answers is not in the document]
Comment (leo-gan) [reply docs/extras/use_cases/apis.ipynb:161]: I need to sleep a little bit more. :nauseated_face: sure

**Pull request merged**